src/main/java/org/sagebionetworks/bridge/services/EnrollmentService.java

```diff
@@ -109,8 +113,10 @@ protected DateTime getWithdrawalDateTime() {
                 .orElseThrow(() -> new EntityNotFoundException(Account.class));
 
         CAN_EDIT_ENROLLMENTS.checkAndThrow(STUDY_ID, studyId, USER_ID, account.getId());
-
-        return enrollmentDao.getEnrollmentsForUser(appId, account.getId());
+        
+        RequestContext context = RequestContext.get();
```

Comment:
If you are not associated to an organization, you are going to see all studies, and that is bad.

Reply:
And in the case of getting studies, what happens if the caller is not in an organization?